contracts.json

```diff
@@ -76,21 +76,21 @@
       "proxyAddress": "0xFAdEDFe89B5A530C0a7f69bE442E190751a13093"
     }
   },
-  "mumbai": {
-    "BridgedPolygonNORI": {
-      "proxyAddress": "0xd768Df6475a853194dB4afF2dDbDC0ea29F05c66"
-    },
+  "amoy": {
     "Removal": {
-      "proxyAddress": "0x8a696730b53925d4118D96C45d53BC1Cf6925bB5"
+      "proxyAddress": "0x1bFD3e01498abf464dD1918Cfb0571716775513C"
     },
     "Certificate": {
-      "proxyAddress": "0xdE41AdFe4F369626925dD989b3ED0893088Fa2a4"
+      "proxyAddress": "0x9A7833C90bB3577730Bbc5d5b669aF63bdB3cd46"
     },
     "Market": {
-      "proxyAddress": "0xE239B41B0B6e8da4E1910fF7FE573629efd5255e"
+      "proxyAddress": "0xfC77384950DB6769e1815c88b26b3eB06B8621Be"
+    },
+    "BridgedPolygonNORI": {
+      "proxyAddress": "0x0"
     },
     "LockedNORI": {
-      "proxyAddress": "0xCdcB43CB7B668F0C1cA04FE4b60DA7F8C62Be393"
+      "proxyAddress": "0x0"
```

Comment:
these need to be in the config based on our type system that I'm not going to overhaul here so I include the stub hex value